mephisto/operations/operator.py

```diff
@@ -245,6 +245,7 @@ def validate_and_run_config_or_die(
             provider.setup_resources_for_task_run(
                 task_run, run_config, shared_state, task_url
             )
+            print(provider.print_banner_on_start(task_run))
```

Comment:
At this point it's likely a little too late for a person to really use this kind of configuration information to make a decision (or potentially change the launch). Realistically you'd probably want to be displaying this just around the `assert_task_args` section.